packages/design-system/src/styles/_reset.scss

```diff
@@ -0,0 +1,344 @@
+/* Core root defaults */
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+html:focus-within {
+  scroll-behavior: smooth;
+}
+
+// Remove all animations, transitions and smooth scroll
+// for people that prefer not to see them
+@media (prefers-reduced-motion: reduce) {
+  html:focus-within {
+    scroll-behavior: auto;
+  }
+
+  *,
+  *::before,
+  *::after {
+    animation-duration: 0.01ms !important;
+    animation-iteration-count: 1 !important;
+    scroll-behavior: auto !important;
+    transition-duration: 0.01ms !important;
+  }
+}
+
+/* Core body defaults */
+
+body {
+  background-color: var(--color-background);
+  color: var(--color-base);
+  font-family: var(--font-body);
+  font-size: var(--font-size-base);
+
+  // 👀 CMS-WDS/scout uses these reset styles
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  line-height: var(--font-line-height-base);
+  margin: 0;
+  min-height: 100vh;
+  text-rendering: optimizeSpeed;
+}
+
+// 👀 CMS-WDS/sep-screener uses this reset style
+// I've seen similar styles in other repos and I'm curious what it's solving
+#main[tabindex='-1']:focus,
+h1[tabindex='-1']:focus {
+  outline: 0;
+}
+*[tabindex='-1']:focus {
+  outline: 0;
+}
+
+/* Typography */
+// ❓ Do we want to enforce max-width rules on type els?
+// I personally feel like doing this would result in more support work for us
+// but it could result in better enforcement of this rule.
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  // `font` is a shorthand property:
+  // https://developer.mozilla.org/en-US/docs/Web/CSS/font
+  // Heading tags inherit all font style rules to match body text;
+  // This ensures semantics are distinct from appearance.
+  font: inherit;
+  // margin applied to top of heading, informed by current margin size
+  margin-block: 1.5em 0;
+}
+
+p {
+  // ❓margin applied to top of text, informed by Chrome UA style - do we want to codify it?
+  margin-block: 1em 0;
+
+  // ❓taken from ./_base.scss ln 329 - do we want to add this to the reset?
+  // CMS-WDS/application @extends this style for their reset:
+  // https://github.cms.gov/CMS-WDS/application/blob/6de8f3ac19e86b7653f0102dcfff7a79241dcb33/app/styles/base/index.scss#L14
+  &:first-child {
+    margin-block-start: 0;
+  }
+}
+
+// These tags will naturally use 700 weight as bold
+// but because some themes have 600 weight, I'm using
+// the CSS variable to adjust bold based on theme
+b,
+strong {
+  font-weight: var(--font-weight-bold);
+}
+
+a {
+  color: var(--link__color);
+  // 📢 We may want to revisit tokens for `text-decoration-style`
+  // ❓Do we have an instance where the underline doesn't match the text color?
```

Comment:
@line47 or @brentschneider know? I can't find any examples like that.

Reply:
I don't think we do have any examples we could I'll ask and try and get some intel on it